webapp/cypress/integration/login/login.spec.js

```diff
@@ -0,0 +1 @@
+beforeEach(() => {})
```

Comment:
to chyba mało potrzebne

Reply:
jak i cały plik

Reply:
wywaliłem cały plik i beforeeach z pliku register